clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -257,7 +257,7 @@ protected synchronized boolean ensureCoordinatorReady(final Timer timer) {
             } else if (coordinator != null && client.isUnavailable(coordinator)) {
                 // we found the coordinator, but the connection has failed, so mark
                 // it dead and backoff before retrying discovery
-                markCoordinatorUnknown();
+                markCoordinatorUnknown("coordinator unavailable");
```

Comment:
Should this be extracted into a constant?

Reply:
I'm ok with it. This is just informational and the literal value is easier to read.